source/extensions/tracers/xray/tracer.h

```diff
@@ -132,6 +132,9 @@ class Span : public Tracing::Span, Logger::Loggable<Logger::Id::config> {
    */
   void log(Envoy::SystemTime, const std::string&) override {}
 
+  void setBaggage(const std::string&, const std::string&) override {}
+  std::string getBaggage(const std::string&) override { return std::string(); }
```

Comment:
implemented in the header for xray simply because it seems thats the convention for this tracer

Reply:
(as opposed to the .cc)